openlibrary/plugins/upstream/borrow.py

```diff
@@ -123,19 +123,25 @@ def POST(self, key):
         # to result if `open`, redirect to bookreader
         response = lending.get_availability_of_ocaid(edition.ocaid)
         availability = response[edition.ocaid] if response else {}
+        archive_url = 'https://archive.org/stream/' + edition.ocaid + '?ref=ol'
```

Comment:
In the future, this should probably use `lending.config_ia_domain` var from openlibrary.yml

Reply:
won't fix